orchestra/utils/task_lifecycle.py

```diff
@@ -37,6 +41,40 @@ class AssignmentPolicyType(object):
     REVIEWER = 'reviewer'
 
 
```

Comment:
this duplicates some code in orchestra/communication/staffing.py:handle_staffing_response, which also calls `assign_task`. 

maybe make a `mark_worker_as_winner` get called by `assign_task` and `reassign_task` and then don't declare the winner int he `handle_staffing_response` function, but pass a default communication preference or staffing inquiry to use?